src/config.json

```
@@ -0,0 +1 @@
+{"username": "Username", "password": "Password", "is_main": true, "targets_list": []}
```

Since we have placeholders now, I say we make each of these `None`, `True`, and `[]`
```suggestion
{"username": None, "password": None, "is_main": True, "targets_list": []}
```

---

None is not valid json.

---

Ah yeah got mixed up that works